backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/testcontroller/ShopControllerTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.testcontroller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gpay.virtualqueue.backendservice.controllers.ShopController;
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+import com.google.gpay.virtualqueue.backendservice.service.ShopService;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(ShopController.class)
+@ContextConfiguration(classes = ShopController.class)
+public class ShopControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private ShopService shopService;
+
+    ObjectMapper om = new ObjectMapper();
+
+    @Test
+    public void testCreateShop() throws Exception {
+
+        Shop shop1 = new Shop("uid1", "name", "shop1", "address", "1111111111", "type");
+        String inputJson = om.writeValueAsString(shop1);
+        
+        when(shopService.createShop(any(Shop.class))).thenReturn(1);
+        MvcResult result = mvc.perform(post("/shop").contentType(APPLICATION_JSON).content(inputJson))
```

Comment:
Nit. Add newline above.

when(shopService.createShop(any(Shop.class))).thenReturn(1); is a part of arrange block.